### ISSUE TITLE
Weird rendering of tooltips

### ISSUE THREAD

**Issue by IdrisQe** [toyhippogriff/AppleJuiceSkin#2]
![image](https://user-images.githubusercontent.com/40530630/45933320-115db980-bf3f-11e8-91de-434256ddbe24.png)
For some reason in the latest version the half-points of thirst and half-points of hydration render backward. Of note, I have the latest AppleCore and AppleSkin, as well as Optifine, Foamfix, SmoothFont, and VanillaFix, all of which could theoretically affect rendering of the tooltips although I don't think any of them really do, especially not in a way that should cause this very specific issue.

**Comment by naqaden** [comment]
This bugged me too. I notice this is the way the art is oriented for the HUD, which fills right-to-left. It doesn't look right for tooltips that render left-to-right.

**Comment by toyhippogriff** [comment]
Yeah, this is just what the texture looks like. I suppose I could mirror the texture for tooltips in the next update.
